src/measure_theory/vitali_caratheodory.lean

```diff
@@ -420,9 +420,9 @@ end
 
 /-! ### Vitali-Carathéodory theorem -/
 
-/-- Vitali-Carathéodory theorem: given an integrable real function `f`, there exists an integrable
-function `g > f` which is lower semicontinuous, with integral arbitrarily close to that of `f`.
-This function has to be `ereal`-valued in general. -/
+/-- **Vitali-Carathéodory Theorem**: given an integrable real function `f`, there exists an 
```

Comment:
Same questions as above: there are other versions of the Vitali-Carathéodory Theorem below.

Reply:
I think I got all of them now